internal/utils/comparators/daemonsets/comparator.go

```diff
@@ -9,61 +9,61 @@ import (
 )
 
 //AreSame compares daemonset for equality and return true equal otherwise false
```

Comment:
NIT: docs not updated, now func return also string value with short explanation, what actually not the same